split.Console/Models/Card.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+namespace split.Console.Models
+{
+    public struct Card
+    {
+        private SuitType Suit;
+        private RankType Rank;
+
+        public Card(SuitType suit, RankType rank)
+        {
+            Suit = suit;
+            Rank = rank;
+        }
+
+        public SuitType GetSuit()
+        {
+            return Suit;
+        }
+
+        public RankType GetRank()
+        {
+            return Rank;
+        }
+
+        public int GetValue()
+        {
+            switch (Rank)
+            {
```

Comment:
I am not sure how necessary this ```switch``` is since ```Enum``` types are [assigned](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum) values by default. In my implementation, I created a [method](https://github.com/Code-Oops/split/blob/feature/composition/split.Console/Models/Blackjack/BlackjackGame.cs#L60) that determines if the card is a face card.